src/main/resources/application.conf

```diff
@@ -379,6 +379,13 @@ mantis {
     # if: block_number - current_best_block_number > max-queued-block-number-ahead
     # then: the block will not be queued (such already queued blocks will be removed)
     max-queued-block-number-ahead = 10
+
+    # Maxium number of blocks, after which block hash from NewBlockHashes packet is considered ancient
+    # and peer sending it is blacklisted
```

Comment:
What does _number of blocks_ mean? Is it how old a hash is in relation to our current known best block number?

Reply:
Yes, exactly it is it, in other words `distance (in number of blocks) of received hash from our current best block`